venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,362 @@
+import argparse
+from re import sub
+import openreview
+from tqdm import tqdm
+import csv
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+
+# Posting submission invitation 
+submission_invitation = openreview.Invitation(
+    id = "aclweb.org/ACL/2022/Conference/-/Submission",
+    signatures = ["aclweb.org/ACL/2022/Conference"],
+    readers = ["everyone"],
+    invitees=["everyone"],
+    reply={
+        "readers":{"values-copied":["aclweb.org/ACL/2022/Conference/Program_Chairs","aclweb.org/ACL/2022/Conference"]},
+        "writers":{"values-copied":["aclweb.org/ACL/2022/Conference"]},
+        "signatures":{"values-copied":["aclweb.org/ACL/2022/Conference"]},
+        "content":{
+            "paper_link": {
+                "description": "Provide the link to your previous ACL submission",
+                "value-regex": "http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+",
+                "required": True,
+                "order": 10
+                },
+            "paper_type": {
+                "description": "Select if your paper is short or long",
+                "value-radio": [
+                "short",
+                "long"
+                ],
+                "required": True,
+                "order": 11
+                },
+            "track": {
+                "description": "Select the track that best fits your submission.",
```

Comment:
could you format the dictionaries so they are easier to read? add indentation should do the trick.